basculechecks/capabilitiesvalidator.go

```diff
@@ -61,6 +61,7 @@ type EndpointChecker interface {
 // pulls the Authentication object from a context before checking it.
 type CapabilitiesValidator struct {
 	Checker  EndpointChecker
+	Key      []string
```

Comment:
Should we rename this to `KeyPath`? or add a comment explaining that it's the path to follow to get the capabilities?